src/language/en-GB/en-GB.plg_api_reports.ini

```diff
@@ -0,0 +1 @@
+PLG_REPORTS_API_REPORT_NAME_MISSSING="Please enter report name"
```

Comment:
@monikaatvowel  either use  PLG_REPORTS_API this or PLG_API_REPORTS this as a prefix

Reply:
changed to PLG_API_REPORTS